test/transpiler/transpiler_for_test.ts

```diff
@@ -0,0 +1,149 @@
+import {expect} from 'chai';
+import Transpiler from '../../src/transpiler';
+
+describe('Transpiler', () => {
```

Comment:
Add a test for for-loops without braces.

Reply:
There is already a for-loop test without braces.